tests/test_core.py

```diff
@@ -929,6 +930,8 @@ def test_scaper_instantiate():
     jam = sc._instantiate(disable_instantiation_warnings=True)
     regjam = jams.load(REG_JAM_PATH)
     _compare_scaper_jams(jam, regjam)
+    ann = jam.search(namespace='scaper')[0]
+    assert ann.sandbox.scaper['sr'] == sc.sr
```

Comment:
This test seems little ad hoc to me: it's checking whether the sr field belonging to the scaper object matches the sr field in the instantiated annotation sandbox. But no other fields are compared (e.g. `ref_db`). It might generally be a good idea to add a set of tests to ensure all annotation sandbox fields match their corresponding object parameters, but right now this seems a little ad hoc?

Reply:
Yeah I didn't see that the sr was being compared in the _compare_scaper_jams function so I was being explicit, but if we fix that function we can remove that test.

Reply:
Yes, we should fix the function and remove these two lines.